requirements.txt

```diff
@@ -1,2 +1,6 @@
-mock==2.0.0
-termcolor==1.1.0
+coverage
+flake8
+mock
+nose
+termcolor
+tox
```

Comment:
I thought about that too. However, we have been installing these package manually without the requirements file from the very first day. And up to now, we're still good.